Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp

```diff
@@ -231,12 +231,12 @@ void GeneralWidget::AddDescriptions()
 {
 // We need QObject::tr
 #if defined(_WIN32)
-  static const char TR_BACKEND_DESCRIPTION[] =
-      QT_TR_NOOP("Selects what graphics API to use internally.\nThe software renderer is extremely "
-                 "slow and only useful for debugging, so you'll want to use either Direct3D or "
-                 "OpenGL. Different games and different GPUs will behave differently on each "
-                 "backend, so for the best emulation experience it's recommended to try both and "
-                 "choose the one that's less problematic.\n\nIf unsure, select OpenGL.");
+  static const char TR_BACKEND_DESCRIPTION[] = QT_TR_NOOP(
+      "Selects what graphics API to use internally.\n\nThe software renderer is extremely "
+      "slow and only useful for debugging, so you'll want to use either OpenGL, Direct3D, "
+      "or Vulkan. Different games and different GPUs will behave differently on each "
+      "backend, so for the best emulation experience it's recommended to try each and "
```

Comment:
AFAIK all major HIGs recommend not to use casual language like "it's" and suggest "it is" instead. Same with "you'll".

Reply:
Yeah, IIRC a lot of descriptions in Dolphin use contractions :/

I’ll fix it here